web/shared/src/typeahead.ts

```diff
@@ -308,7 +320,7 @@ export function triage<T>(
     };
 }
 
-export function sort_emojis<T extends Emoji>(objs: T[], query: string): T[] {
+export function sort_emojis<T extends Emoji | EmojiPillData>(objs: T[], query: string): T[] {
```

Comment:
Ideally this would just be `EmojiPillData` once I consolidate the types a bit better.